### ISSUE TITLE
Dedicated connection string for administrative access

### ISSUE THREAD

**Issue by twenzel** [Particular/NServiceBus.RabbitMQ#929]
Please provide an option to define a dedicated connection string which is used for creating the topology (queues, exchanges..).

As mentioned in your docs, the Installer feature should not be used in production because of permission settings.
This option will help prevent a security issue but using your topology creation at the same time.

**Comment by helenktsai** [comment]
Hi @twenzel, thanks for your request. I don't know when this will be addressed, but it has been added to our backlog for review.

**Comment by twenzel** [comment]
@ramonsmits You're right. The least-privilege principle is getting hurt with this approach.
Ideally the setup can be done outside the process, maybe upfront within the deployment process.
Some kind of CLI would be awesome.
